Validate `HIP_TARGETS` format and allow negative values (#2481)
IntelPython/dpnp: CHANGELOG.md

### Changed

* Adjusted the `pre-commit` configuration to run autoupdate weekly [#2479](https://github.com/IntelPython/dpnp/pull/2479)

> * Improved validation of `--target-hip` build option to only accept a gfx-prefixed value [#2479](https://github.com/IntelPython/dpnp/pull/2479)

Reviewer: ```suggestion
* Improved validation of `--target-hip` build option to only accept a gfx-prefixed value [#2481](https://github.com/IntelPython/dpnp/pull/2481)
```

> * Improved `--target-hip` handling to accept only gfx-prefixed values and allow disabling HIP build [#2479](https://github.com/IntelPython/dpnp/pull/2479)

Reviewer: ```suggestion
* Improved validation of `--target-hip` build option to only accept a gfx-prefixed value [#2479](https://github.com/IntelPython/dpnp/pull/2479)
```
